cdap-api/src/main/java/co/cask/cdap/api/common/Scope.java

```diff
@@ -32,7 +32,7 @@
   /**
    * Private constructor to force using the enum values.
```

Comment:
It is private. All enum constructor are private, that's why no need to declare it as "private" (checkstyle says redundant modifier).